backend/apps/api/src/routes/image.py

```
@@ -1,13 +1,125 @@
+            "size": img.size,
+            "url": url,
+            "created_at": img.created_at.isoformat()
+        })
+    
+    return {"images": result, "count": len(result)}
+
+
+@router.get("/images/user/{user_id}")
+async def list_user_images(
+    user_id: int = Path(..., description="사용자 ID"),
+    db: Session = Depends(get_db),
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)